app/assets/javascripts/MathJax/mathjax_helpers.js

```diff
@@ -3,3 +3,25 @@
 function reloadDOM() {
     MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
 }
+
+function updateAnnotationPreview(){
+    var newContent = document.getElementById('new_annotation_content').value;
+
+    var preview_paragraph = document.getElementById('annotation_preview');
```

Comment:
Identifier 'preview_paragraph' is not in camel case.<br>Mixed double and single quotes.